src/compas_cgal/straight_skeleton_2.py

```diff
@@ -1,14 +1,47 @@
 import numpy as np
+from compas.datastructures import Graph
 from compas.geometry import Polygon
 from compas.geometry import normal_polygon
 from compas.tolerance import TOL
 
 from compas_cgal._cgal import straight_skeleton_2
 
-from .types import PolylinesNumpy
 
+def graph_from_skeleton_data(points, indices, edges, edge_types) -> Graph:
```

Comment:
since you annotate the output, perhaps you could also annotate the input?

Reply:
i am new to using types for in- and outputs. but I hope this is what you meant now. ;)